### PR TITLE
[l10n] Add Danish (daDK) locale thanks to @kasperfilstrup

### PR DESCRIPTION
**Pull request by alexfauquette**
see issue #3796

### CODE REVIEW

File: packages/grid/_modules_/grid/locales/daDK.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import { getGridLocalization, Localization } from '../utils/getGridLocalization';
+
+const daDKGrid: Partial<GridLocaleText> = {
+  // Root
+  noRowsLabel: 'Ingen rækker',
+  noResultsOverlayLabel: 'Ingen resultater',
+  errorOverlayDefaultLabel: 'Der skete en fejl.',
+
+  // Density selector toolbar button text
+  toolbarDensity: 'Density',
```
Comment (flaviendelangle) [inline packages/grid/_modules_/grid/locales/daDK.ts:12]: According to Google Translate, this is not Danish
Same for others below.
Could @kasperfilstrup indicate the untranslated one and either translate them or comment them ?

Comment (alexfauquette) [reply packages/grid/_modules_/grid/locales/daDK.ts]: Good catch. I commented on lines that are in English

**Review by flaviendelangle — APPROVED** [review]

File: packages/grid/_modules_/grid/locales/daDK.ts (hunk 1 above)
Comment (alexfauquette) [reply packages/grid/_modules_/grid/locales/daDK.ts]: @kasperfilstrup if you want to modify translation to this PR, the easiest way is to make suggestions on the [change files](https://github.com/mui-org/material-ui-x/pull/3800/files)

Screenshot from 2022-01-31 14-09-17

**Review by cherniavskii — APPROVED** [review]
Would be great to have https://github.com/mui-org/material-ui-x/issues/3211 updated after merging this PR :)

**Comment by alexfauquette** [comment]
@cherniavskii We do not have to care about #3211 the update is made automatically by the CI when there is a push on master :)

**Comment by cherniavskii** [comment]
@alexfauquette oh, that's cool :)

**Comment by flaviendelangle** [comment]
@alexfauquette can we merge that one ?

**Pull request merged**